src/round.rs

```diff
@@ -176,4 +324,113 @@ mod tests {
         assert_eq!(dt.trunc_subsecs(0).nanosecond(), 1_000_000_000);
         assert_eq!(dt.trunc_subsecs(0).second(), 59);
     }
+
+    #[test]
+    fn test_duration_round() {
+        let dt = Utc.ymd(2016, 12, 31).and_hms_nano(23, 59, 59, 1_75_500_000);
+
+        assert_eq!(
+            dt.duration_round(Duration::milliseconds(10)).unwrap().to_string(),
+            "2016-12-31 23:59:59.180 UTC"
+        );
+
+        let dt = Utc.ymd(2012, 12, 12).and_hms(18, 24, 12);
```

Comment:
Could you add some DTs that exactly test the borders? e.g. `Utc.ymd(2012, 12, 12).and_hms_milis(12, 30, 30, 500)` and `Utc.ymd(2012, 12, 12).and_hms_milis(11, 29, 29, 499)`?